src/main/java/me/rigamortis/seppuku/impl/module/combat/NoCrystalModule.java

```diff
@@ -236,6 +266,12 @@ private boolean valid(BlockPos pos) {
         final Block south = mc.world.getBlockState(pos.add(0, 0, 1)).getBlock();
         final Block east = mc.world.getBlockState(pos.add(1, 0, 0)).getBlock();
         final Block west = mc.world.getBlockState(pos.add(-1, 0, 0)).getBlock();
+        
+        for (Entity entity : mc.world.getEntitiesWithinAABBExcludingEntity(null, new AxisAlignedBB(pos))) {
+            if (entity instanceof EntityLivingBase) {
```

Comment:
Some non living entities (such as falling blocks and minecarts) can block placing

Reply:
You can check if the entity has `preventEntitySpawning` set to true, or preferably use `mc.world.checkNoEntityCollision(bb, entity)`